src/Umbraco.Web/Editors/EntityController.cs

```diff
@@ -382,6 +383,53 @@ public IEnumerable<EntityBasic> GetChildren(int id, UmbracoEntityTypes type)
             return GetResultForChildren(id, type);
         }
 
+        public PagedResult<EntityBasic> GetPagedChildren(
+            int id,

```

Comment:
Yup that's definitely it, we are passing in the string "all-members" which is a special Id, since members don't have children this method doesn't make a ton of sense but we can make it work. In 7.6 we are using special action selectors to determine which overload to use since in 7.6. we have UDIs which are also strings. So i can use this same technique for this too and then do a special check against the "all-members" id